kubernetes-client/1.3.76.wso2v1/pom.xml

```diff
@@ -0,0 +1,145 @@
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-api</artifactId>
+            <version>${kubernetes-api.version}</version>
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Vendor>WSO2, Inc.</Bundle-Vendor>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            io.fabric8.kubernetes.*;version="${project.version}";-split-package:=merge-first,
+                        </Export-Package>
+                        <Private-Package>
+                            io.fabric8.kubernetes.client.internal.*
+                        </Private-Package>
+                        <Import-Package>
+                            !io.fabric8.kubernetes.client.*,
+                            com.fasterxml.jackson.annotation;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.core;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.core.type;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind.annotation;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind.node;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.dataformat.yaml;version="[2.7.0,3.0.0)",
+                            com.mifmif.common.regex;resolution:=optional,
+                            com.squareup.okhttp;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.logging;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.ws;version="[2.7.2,3.0.0)",
+                            io.fabric8.openshift.client.dsl;resolution:=optional;version="[1.3.0,2.0.0)",
+                            io.fabric8.openshift.api.model;resolution:=optional;version="[1.0,2)",
+                            io.fabric8.openshift.client;resolution:=optional;version="[1.3.0,2.0.0)",
+                            io.fabric8.utils;resolution:=optional;version="[2.2.0,3.0.0)",
+                            io.fabric8.utils.ssl;resolution:=optional;version="[2.2.0,3.0.0)",
+                            javax.net,
+                            javax.net.ssl,
+                            javax.security.auth.x500,
+                            javax.tools;resolution:=optional,
```

Comment:
fixed in https://github.com/wso2/orbit/commit/3119064e24811ce44ddeb10f16d677a572e8aaa3